tests/test_server/test_search_server.py

```diff
@@ -33,15 +33,14 @@ def search_client(
     """Fixture to create a client for mining_server."""
 
     sbiobert_model_inst = Mock()
-    sbiobert_model_class = Mock()
-    sbiobert_model_class.return_value = sbiobert_model_inst
     sbiobert_model_inst.preprocess.return_value = "hello"
```

Comment:
Maybe we can change the variable name to something different? Might be confusing to still use `sbiobert_model..`

Reply:
Renamed the variable as `fake_embedding_model` in 7a66438.